packages/flutter/lib/src/material/calendar_date_picker.dart

```diff
@@ -27,11 +27,20 @@ import 'theme.dart';
 
 const Duration _monthScrollDuration = Duration(milliseconds: 200);
 
-const double _dayPickerRowHeight = 42.0;
+// Current M2 implementation is not compliant with the M2 specification.
+// Instead of a 42 pixels row height it should be 40 with a 2 pixels inner padding.
```

Comment:
```suggestion
// Instead of a 42 pixels row height it should be 40 with a 2 pixels inner padding.
// There is no plan to fix this since M2 is being deprecated.
```

Reply:
(Is it correct to call M2 being deprecated? I did not participate in the decision making.)

Reply:
AFAIK, there was no decision to deprecate M2. We usually focus on M3, but many PRs also fix M2 issues when possible.